src/SDKs/IotHub/IotHub.Tests/Helpers/IotHubTestUtilities.cs

```diff
@@ -1,24 +1,36 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
+﻿//
+// Copyright (c) Microsoft.  All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
```

Comment:
@ssankar1984 why change the header and use the apache license?
We have adopted to use MIT license, please retain the previous license header